src/pages/Gift/Mailer/Form.tsx

```diff
@@ -15,11 +16,18 @@ export default function Form({ classes = "" }) {
   const { handleSubmit, reset } = useFormContext<FV>();
```

Comment:
```suggestion
  const { handleSubmit, reset, formState:{ isSubmitting } } = useFormContext<FV>();
```
you could use this state unless there's a reason for separate `isSending` state ?